shared/stats/src/index.mjs

```diff
@@ -44,14 +61,50 @@ export async function getStats (modulePath) {
   // Modules bundled
   const modules = stats?.[base] ?? {}
 
+  const moduleCount = Object.keys(modules).length
+
   // Modules total size
   const total = Object.values(modules)
     .map(({ rendered }) => rendered)
     .reduce((total, rendered) => total + rendered, 0)
 
-  return { total, modules }
+  return { total, modules, moduleCount }
+}
+
+/**
+ * Returns file sizes of key files
+ *
+ * @returns {Promise<{[key: string]: string | number | any[] | FileSizeObject}>} - File names and size
```

Comment:
My best attempt at a return type. `filesize` exports the `filesize` function, but doesn't export what that function returns, so I've hacked it in here.